js/ngDialog.js

```diff
@@ -7,7 +7,7 @@
 (function (root, factory) {
     if (typeof module !== 'undefined' && module.exports) {
         // CommonJS
-        module.exports = factory(require('angular'));
+        module.exports = require('angular');
```

Comment:
This is not right.. and the `factory` should exist.

@DevinClark I assume you are using angular 1.3, in which (for a very stupid reason)  they changed the module, so now it doesn't export anything.

Check the discussion and possible workarounds [here](https://github.com/angular/angular.js/issues/9699) or [here](http://stackoverflow.com/questions/26452359/updated-to-angular-1-3-0-no-longer-working).

`ngDialog` itself is fine, I'm using it in browserified projects without any issues.